.github/workflows/templates/layout.yml.jinja

```
@@ -22,8 +22,14 @@ run-name: "<{ workflow_name }> (${{ github.event_name == 'pull_request' && forma
 <%- block on %>
 
 on:
-  push: {}
-  pull_request: {}
+  push:
+    types:
+      - labeled
+      - unlabeled
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
```

```suggestion
  push:
    types:
      - labeled
      - unlabeled
  pull_request:
    types:
      - labeled
      - unlabeled
```
I get the labeled/unlabeled in `pull_request`, but how to we label/un-label a push?